### ISSUE TITLE
len(pats) will throws IndexError. if using pats = Patent.objects.filter(title="tennis", issue_date="2010-01-01->2010-02-27")

### ISSUE THREAD

**Issue by grimmer0125** [parkerhancock/patent_client#45]
I modified the example in https://patent-client.readthedocs.io/en/latest/index.html#super-quick-start
```
Patent.objects.filter(title="tennis", issue_date="2010-01-01->2010-12-31")`
```
to 
```
pats = Patent.objects.filter(title="tennis", issue_date="2010-01-01->2010-02-27")
```
Then
```
len(pats) > 10 # throw error in  self.num_results = int(soup.find_all("i")[1].find_all("strong")[-1].text)
```

**Comment by parkerhancock** [comment]
Thank you for your interest in Patent Client! I don't think this is an issue any longer. The Readme assigns the result to the pats variable. Let me know if this isn't right, and I'm happy to revisit.

**Comment by grimmer0125** [comment]
Hi @parkerhancock , 

My previous description is not clear. I tried the same query condition on `patft` and there is `one` search result. The special thing is that it will automatically redirect to the patent page. If using a python script to query, the result is a small html page contains `single document` this word and the patent URL listed inside but it is not the predicted result page. Therefore, it will throw an exception here
```
            self.num_results = int(soup.find_all("i")[1].find_all("strong")[-1].text)
```
I made some workaround code here which is not beautiful and just as a reference, https://github.com/grimmer0125/patent_client/commit/8943ca970d174ecf8aad10669a95ceddd75f5e13#diff-b6dbafaca666db361cb7b2b2266719b822e2dcecd53671c5f29dcf92a6590799R158
